src/app/backend/handler/apihandler.go

```diff
@@ -532,27 +533,27 @@ func CreateHTTPAPIHandler(iManager integration.IntegrationManager, cManager clie
 			Writes(persistentvolumeclaim.PersistentVolumeClaimDetail{}))
 
 	apiV1Ws.Route(
-		apiV1Ws.GET("/crd").
+		apiV1Ws.GET("/customresourcedefinition").
```

Comment:
Looks like you rolled back some changes.

Reply:
Yes, I renamed the route so that it's simply the actual `kind` of the resource, which is what will be used in the `Resource.crd` enum.
However, I've reverted it again and simply hardcoded the string literal in the one place where I needed the actual kind, not the short form (https://github.com/kubernetes/dashboard/pull/4159/commits/cdb6900b4acbc273fe536fbc941ebfb776275ad8#diff-d4c5451b6048107c5a2560cba1cf2063L41).